CSS/styleGuide.css

```diff
@@ -30,6 +26,7 @@ html {
   --background-color: rgb(240 240 240);
   --text-color: #353434;
   --white-color: white;
+  --white2-color: floralwhite;
   --greenishWhite: rgb(225, 228, 226);
   --black-color: black;
```

Comment:
you don't need to define so many colors, as it's hard from a UX perspective to pull off so many colors, as most websites need at most 3-4 colors, and if you want to create shades of the colors there is a better approach, read More :-

https://stackoverflow.com/questions/40010597/how-do-i-apply-opacity-to-a-css-color-variable

Reply:
Thank you for the suggestion. I will read this.